src/Modules/ExteriorPowers/Generic.jl

```diff
@@ -1,14 +1,14 @@
 # We need to cache eventually created exterior powers.
-@attr Dict{Int, Tuple{T, <:Map}} function _exterior_powers(F::T) where {T<:ModuleFP}
-  return Dict{Int, Tuple{typeof(F), Map}}()
+@attr Dict{Int, Tuple{typeof(F), MapFromFunc}} function _exterior_powers(F::ModuleFP)
+  return Dict{Int, Tuple{typeof(F), MapFromFunc}}()
```

Comment:
```suggestion
@attr Dict{Int, Tuple{typeof(F), <:MapFromFunc}} function _exterior_powers(F::ModuleFP)
  return Dict{Int, Tuple{typeof(F), MapFromFunc}}()
```
This might be a bit too restrictive. Maybe, one day we change the return type. And they are both abstract, so no harm is done staying with just `Map` right?

Reply:
Tuples are invariant in julia, thus the `<:` shouldn't be there. 
As all of this is purely internal, we can change that to Map back anytime we want. But making it as concrete as possible helps type inference massively. In this particular case, it can use some of the annotations in the `MapFromFunc` code in Hecke